client/emtpy-state-table/list.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import wcpayTracks from 'tracks';
+
+export const EmptyStateTableHeaders = [
+	{
+		text: 'Date',
+		classNames: 'is-sorted is-left-aligned',
+	},
+	{
+		text: 'Type',
+	},
+	{
+		text: 'Amount',
+	},
+	{
+		text: 'Bank account',
+	},
+];
+
+export const EmptyStateList = ( props ) => {
+	const [ isSubmitted, setSubmitted ] = useState( false );
+
+	const handleSetup = () => {
+		setSubmitted( true );
+		wcpayTracks.recordEvent( wcpayTracks.events.CONNECT_ACCOUNT_CLICKED, {
+			// eslint-disable-next-line camelcase
+			wpcom_connection: wcpaySettings.isJetpackConnected ? 'Yes' : 'No',
+		} );
+	};
+
+	return (
+		<div className="empty-state-deposits-list">
+			<div>
+				<img src={ props.listBanner } alt="" />
+			</div>
+			<p className="intro-copy">
```

Comment:
This text isn't i18n.

Reply:
Thanks @moon032! Looks like the `'woocommerce-payments'` text domain was missed on the intro copy text.

Reply:
@adrianduffell Thank you for catching it 👍  Updated in  https://github.com/Automattic/woocommerce-payments/pull/1963/commits/a52ce4a107ef9c24cd4a4ce50319112e45f80cf6

I also removed the overview page on the deposits screen as per Elizabeth in https://github.com/Automattic/woocommerce-payments/pull/1963/commits/d2d767bf1934f2f199ac8114f9823378793940bc